### PR TITLE
My promised PR :)

### PR DESCRIPTION
**Pull request by fangchin**
Dear Speey Sven,

Here is my PR.  I am happy that I can make it on the Christmas Day! :blush:

Please review and if you find everything fine, merge it.  I do hope it will promote even more the use and popularity of `elbencho`! :thumbsup:

Thanks and take care,

Chin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,94 +1,40 @@
...
+A single Docker container hosted on a GIGABYTE R281-NO0-00.  It is the
+only container hosted on the server while the tests were conducted.
+
+* 2x12-core Intel(R) Xeon(R) Gold 6146 CPU @ 3.20GHz
+* 384GB RAM
+* 8x NVMe 3.84TB SSDs in Redundant Arrays of Independent Disks
+  (RAID)-0 (Intel VROC) [HGST Ultrastar SN200 Series NVMe SSD]
+* Mellanox ConnectX-5 dual-port 100G NICs
+* CentOS Linux release 7.8.2003 (Core)
+* Filesystem: XFS
```
Comment (breuner) [inline README.md:40]: This commit seems to unintentionally delete the text of the main README.md file and replace it with text that was intended for the README.md in the mtelbencho subdir. Could you please check this?

Comment (fangchin) [reply README.md:40]: My apologies.  Will get more coffee!  I will fix my repo.

Comment (fangchin) [reply README.md]: Or I can cancel the PR and resent one?

**Comment by breuner** [comment]
Thanks a lot for this excellent contribution, Chin!

**Pull request merged**